docs/assets/resources/docker/bashrc-strongbox

```diff
@@ -90,33 +90,33 @@ dockerLocal() {
 }
 
 dockerMvn35Checkout() {
-    dockerCheckout strongboxci/alpine:jdk8-mvn-3.5 "$@"
+    dockerCheckout strongboxci/alpine:jdk8-mvn3.6 "$@"
 }
```

Comment:
Could you also change the function names from `dockerMvn35Checkout` to `dockerMvnCheckout` (and so on)?

Reply:
okay will do.